lib/utils.pm

```diff
@@ -1859,4 +1862,24 @@ sub install_patterns {
     }
 }
 
+sub common_service_action {
+    my ($service, $type, $action) = @_;
+
+    if ($type eq 'SystemV') {
+        if ($action eq 'on') {
+            assert_script_run 'chkconfig ' . $service . ' on';
```

Comment:
assert_script_run 'chkconfig ' . $service . ' on';   == >      assert_script_run 'chkconfig ' . $service . $action;  ?

Reply:
for systemV enable and 'start/stop/restart' use different command. I've put the enable
in ntp to the common_service_action function.